src/client.rs

```diff
@@ -286,9 +289,9 @@ impl RequestBuilder {
             self.headers.set(Accept::star());
         }
         if self.client.auto_ungzip.load(Ordering::Relaxed) &&
-            !self.headers.has::<AcceptEncoding>() &&
-            !self.headers.has::<Range>() {
-            self.headers.set(AcceptEncoding(vec![qitem(Encoding::Gzip)]));
+           !self.headers.has::<AcceptEncoding>() && !self.headers.has::<Range>() {
```

Comment:
There doesn't currently appear to be a way to control the formatting of expressions like this.